promql/promqltest/testdata/aggregators.test

```diff
@@ -380,13 +395,23 @@ load 10s
 	data{test="uneven samples",point="a"} 0
 	data{test="uneven samples",point="b"} 1
 	data{test="uneven samples",point="c"} 4
+	data_histogram{test="histogram sample", point="c"} {{schema:2 count:4 sum:10 buckets:[1 0 0 0 1 0 0 1 1]}}
 	foo .8
 
 eval instant at 1m quantile without(point)(0.8, data)
 	{test="two samples"} 0.8
 	{test="three samples"} 1.6
 	{test="uneven samples"} 2.8
 
+# should produce same results ignoring histograms with info annotations.
+eval_info instant at 1m quantile without(point)(0.8, {__name__=~"^data(_histogram)?$"})
```

Comment:
PromQL regexps are always anchored anyway, so you don't need to do it explicitly here.

Reply:
Yeah, I missed that. We don't need explicit regex here. I'll make the changes.